docker/testnet/supervisord.conf

```diff
@@ -18,7 +18,7 @@ strip_ansi = false
 
 [program:node0]
 command=cheqd-noded start --p2p.laddr tcp://127.0.0.1:26656 --grpc.address 0.0.0.0:9090 --rpc.laddr tcp://0.0.0.0:26657
-directory=/cheqd
+directory=/home/cheqd
 stdout_logfile=/tmp/node0.log
 stderr_logfile=/tmp/node0.log
 
```

Comment:
This whole file is for local network I believe and therefore not needed for our testnet folder.

Reply:
Marking as handled in the new PR for localnet only.